snews.php

```diff
@@ -2188,6 +2051,8 @@ function center() {
 // ROUTES
 addRoute('login', l('login'), 'login');
 addRoute('contact', l('contact'), 'contact');
-addRoute('sitemap', l('sitemap'), 'sitemap');
-addRoute('archive', l('archive'), 'archive');
+
+// LOAD ADDONS
+readAddons();
```

Comment:
I see but then I have a problem with load_lang() in sitemap.
I have to figure out what the sequence should be.